google/cloud/storage/CMakeLists.txt

```diff
@@ -366,5 +369,4 @@ configure_file("${PROJECT_SOURCE_DIR}/google/cloud/config-version.cmake.in"
                "storage_client-config-version.cmake" @ONLY)
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/storage_client-config.cmake"
               "${CMAKE_CURRENT_BINARY_DIR}/storage_client-config-version.cmake"
-              DESTINATION
-              "${CMAKE_INSTALL_LIBDIR}/cmake/storage_client")
+        DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/cmake/Modules")
```

Comment:
This did not seem to work in our CI build:

https://travis-ci.org/GoogleCloudPlatform/google-cloud-cpp/jobs/433548848

Can you take a look?  It is possible that the CI build is misconfigured, but it actually has very little configuration.

Reply:
@coryan 
Hmm. I wonder if it needs to install with prefix=/usr instead of /usr/local/. 
Looks like the ubuntu cmake package has the modules in /usr/share/cmake https://packages.ubuntu.com/bionic/all/cmake-data/filelist and the log shows local. 

Maybe we have to add -DCMAKE_INSTALL_PREFIX=/usr to build-docker.sh?
But if that's the case I'm not entirely sure how it worked before.

Reply:
I think it might be the `Modules`:  CMake:

https://cmake.org/cmake/help/v3.8/command/find_package.html?highlight=i

searches in `<prefix>/share/cmake`, but not in `<prefix>/share/cmake/Modules` for installed packages.

Reply:
Hmm. but I dont have any files in that dir from any other packages on my system.
```
# ls -al /usr/share/cmake/
total 116
drwxr-xr-x.   7 root root   7 Sep  8 01:11 ./
drwxr-xr-x. 245 root root 246 Sep 16 01:48 ../
drwxr-xr-x.  17 root root  18 Jan  4  2018 Help/
drwxr-xr-x.  12 root root 429 Sep 15 03:54 Modules/
drwxr-xr-x.   4 root root  13 Jan  4  2018 Templates/
drwxr-xr-x.   2 root root   4 Apr  1 22:06 bash-completion/
drwxr-xr-x.   2 root root   3 Jan  4  2018 include/
```

I will try out the install-test on my local machine and compare it with travis tomorrow.